response_operations_ui/templates/social-view-case-details.html

```diff
@@ -16,39 +16,29 @@ <h1 class="jupiter u-mt-m" id="case_ref">{{attributes.Reference}}</h1>
         <section class="social-case-section u-mt-l">
             <h2 class="neptune">Property address</h2>
             <dl class="survey-info info-panel" id="case-attributes">
-                {% if attributes.Prem1 %}
-                <dt class="case-address__prem" id="case-prem1">
-                    {{attributes.Prem1}}
+                {% if attributes.ADDRESS_LINE1 %}
```

Comment:
Instead of having to maintain this for every change in a sample file, perhaps we can just have a for-loop?

```jinja2
<dl class="survey-info info-panel" id="case-attributes">
{% for name, attribute in attributes.iteritems() %}
  <dt class="case-address__{{ name|lower }}" id="case-{{ name|lower|replace('_', '-') }}">
    {{ attribute }}
  </dt>
{% endfor %}
</dl>
```

Reply:
Good idea but then we'd be passing data to the html that doesn't need to be displayed to the user, any idea the best way to filter the data that doesn't need to be presented?

Reply:
If we have a list of attribute keys that we only want to use then that can be passed in as a variable and used in the loop instead:

```python
...
return render_template(attributes=sample_attributes['attributes'], 
                       list_of_attributes=["ADDRESS_LINE1", "ADDRESS_LINE2", ...])
```

```jinja2
<dl class="survey-info info-panel" id="case-attributes">
{% for name in list_of_attributes %}
  <dt class="case-address__{{ name|lower }}" id="case-{{ name|lower|replace('_', '-') }}">
    {{ attributes["name"] }}
  </dt>
{% endfor %}
</dl>
```